lib/cc/analyzer/container.rb

```diff
@@ -60,7 +60,7 @@ def run(options = [])
           duration = timeout
           @listener.timed_out(container_data(duration: duration))
         else
-          sleep 0.01 until !t_out.alive? && !t_err.alive?
+          t_out.join || t_err.join
```

Comment:
Is it possible for one of these to finish before the other? Imagining if stderr closes but stdout is still flushing? Is that possible?